src/services/api.service.ts

```diff
@@ -78,25 +80,26 @@ export class APIService {
   private wrapRequest<T>(request: Observable<T>): Observable<T> {
     const timeout = this.config.get('CLIENT_TIMEOUT', 15000);
     let attempts = 0;
-    return request
-      .timeout(timeout)
```

Comment:
import 'rxjs/add/operator/timeout'; does not fix it ?

Reply:
@tdesvenain using `import 'rxjs/add/operator/timeout';` instead of `import { timeout } from 'rxjs/operators';` seems identical, and we get the same error.